css-borders-4/Overview.bs

```diff
@@ -96,11 +89,239 @@ Line Colors: the 'border-color' properties</h3>
+	Name: border-top-width, border-right-width, border-bottom-width, border-left-width, border-block-start-width, border-block-end-width, border-inline-start-width, border-inline-end-width
+	Value: <<line-width>>
+	Initial: medium
+	Applies to: all elements except [=ruby base containers=] and [=ruby annotation containers=]
+	Inherited: no
+	Logical property group: border-width
+	Percentages: N/A
+	Computed value: absolute length, [=snapped as a border width=]; zero if the border style is ''border-style/none'' or ''border-style/hidden''
+	Animation Type: by computed value
+	</pre>
+
+	The logical properties 'border-block-start-width', 'border-block-end-width',
+	'border-inline-start-width', and 'border-inline-end-width' correspond to the
+	'border-top-width', 'border-bottom-width', 'border-left-width',
+	and 'border-right-width' properties.
+	The mapping depends on the element's 'writing-mode', 'direction',
+	and 'text-orientation'.
+
+	<pre class="propdef shorthand">
+	Name: border-block-width, border-inline-width
+	Value: <<'border-top-width'>>{1,2}
+	</pre>
+
+	These two <a>shorthand properties</a> set the
+	'border-block-start-width' &amp; 'border-block-end-width'
+	and
+	'border-inline-start-width' &amp; 'border-inline-end-width',
+	respectively.
+	The first value represents the <a>start</a> edge width,
+	and the second value represents the <a>end</a> edge width.
+	If only one value is given,
+	it applies to both the <a>start</a> and <a>end</a> edges.
+
+<h3 id="border-shorthands" oldids="the-border-shorthands">
+Border Shorthand Properties</h3>
+
+	<pre class="propdef shorthand">
+	Name: border-top, border-right, border-bottom, border-left, border-block-start, border-block-end, border-inline-start, border-inline-end
+	Value: <<line-width>> || <<line-style>> || <<color>>
+	Initial: See individual properties
+	Applies to: all elements except [=ruby base containers=] and [=ruby annotation containers=]
+	Inherited: no
+	Percentages: N/A
+	Computed value: see individual properties
+	Animation Type: see individual properties
+	</pre>
+
+	The logical properties 'border-block-start', 'border-block-end',
+	'border-inline-start', and 'border-inline-end' correspond to the
+	'border-top', 'border-bottom', 'border-left', and 'border-right' properties.
+	The mapping depends on the element's 'writing-mode', 'direction',
+	and 'text-orientation'.
+
+	<pre class="propdef shorthand">
+	Name: border-block, border-inline
+	Value: <<'border-block-start'>>
+	</pre>
+
+	These two <a>shorthand properties</a> set the
+	'border-block-start' &amp; 'border-block-end'
+	or
+	'border-inline-start' &amp; 'border-inline-end',
+	respectively,
+	both to the same style.
+
+	<!-- intentionally cannot set two sides independently;
+		see discussion in https://github.com/w3c/csswg-drafts/issues/3519 -->
+
+<h3 id="logical-shorthand-keyword">
+Four-Directional Shorthand Properties: the 'logical' keyword for 'border-width', 'border-style', and 'border-color'</h3>
```

Comment:
Drop this section, as this issue is being worked out in css-logical. Once that stabilizes, we can add relevant bits of spec text; but probably there will be a central definition that we can link to for these types of shorthands, not one that needs to be copied from spec to spec.

Reply:
Ok, removed.